src/Elm/Syntax/Node.elm

```diff
@@ -44,7 +44,7 @@ empty a =
     Node Range.empty a
 
 
-{-| Combine two nodes, constructing a new node which will have the outer most range of the child nodes
+{-| ParserWithComments two nodes, constructing a new node which will have the outer most range of the child nodes
```

Comment:
I think this was unintended :joy:

Reply:
Thanks for being so thorough with the review! <3 Fixed.